src/main/java/org/spongepowered/common/config/category/TileEntityActivationModCategory.java

```diff
@@ -33,15 +33,16 @@
 @ConfigSerializable
 public class TileEntityActivationModCategory extends ConfigCategory {
 
```

Comment:
I think there needs to be some general explanation in the config to what this does, maybe a copy of the module description? same for all categories really